meetings/2024-05-15.md

```diff
@@ -0,0 +1,134 @@
+* MdJ: We have people who want both. We should allow people, they're both valid use cases. Whehther there;s an authroization agent or apps edit directly, shoudl we mention just one policy registry (WAC, would be overruled by WG)? Should we have the two we have now and ignore eP's work on it and Inrupt's work on access grants? OIDC spec already mentions UMA. Lots of separate efforts happening in the community. Do we want to update the spec to acknowledge all the things? 
+* eP: I dont think SAI should be included for personal/sentimental reaons. ACP/WAC don't meet certain requirements. 
+* VB: I will post my thoughts on the issue, this is a big change. Introduction of ACP already created many issues for front end developers. We should actively involve Practitioners into this discussion.
+* MdJ: Can we agree as the CG that this is a missing part of the funcionality? There are curretn 6 different ways that are being explored. 
+* TBL: No we shouldn't say that there are 6 different ways of doing something. The only thing that actually works is WAC. There was a push to move the community to ACP but ??? we don't see a way to have ??? implementors have to choose ACP/WAC and clients have to implement both. This is not "the cat is in the bag". We have kittens. But there's only one cat. SAI has ??? lots of issues with SAI.. I can see SolidOS moving from WAC to ACP or WAC to WAC 2.0... 
+* eP: if we have UCRs and proposals that meet them, we can say expected behavior are met in this way. When we say WAC works, I haven't seen how the requirement I mentioned is met. 
+* RG: We need to get ??? we need an STM. 
+* VB: More implementation feedback would be useful so that we have more insight into potential issues, security concerns, and so on, that we should be aware before requiring it on the spec.
+* eP: It'd be useful to have UCR so that they can select the work that addresses their requirements. 
+* TBL: Useful to have different use cases. Before we go down into lots of micro use cases, we focus on high level use cases.
+* VB: +1. 
+* MdJ: We already gave SAI more visibility through hackathon. At CG level we could inform of use cases and present SAI. So people know where to find it, keep protocol as it is, make sure people know where SAI is.
+* eP: It's not about SAI. Do you clearly describe references between the 6 different approaches you have mentioned. Would be good to have a way to find a way to evaluate all of them.
+* RG: Having some discussion (STM) to bring us closer.. even if not to a conclusion. So much info that needs to be digested. 
+
+### CG-DRAFTs PRs
+* VB: Please review: https://github.com/solid/specification/pull/651 and https://github.com/solid/specification/pull/652
+
+---
+
+### Eating our own cookings
+
+URL: https://virginiabalseiro.github.io/solid-ecosystem-monitor/
+
+* eP: CG meetings are one of the few things we do together, instead of scraping the text with minutes we could use the power of Linked Data and maybe even Solid
```

Comment:
```suggestion
* eP: CG meetings are one of the few things we do together. Instead of scraping the text with minutes, we could use the power of Linked Data and maybe even Solid.
```